README.md

```diff
@@ -114,6 +114,8 @@ Dann die `config.json` anpassen nach belieben. Es gibt die folgenden Konfigurati
 * **`server.databaseFile`** Pfad zur Datenbank-Datei, z. B.: `"$FFFFNG_HOME/ffffng.sqlite"`
 * **`server.peersPath`** Verzeichnis unter dem die `fastd` Key-Files angelegt werden, z. B.: `"$FFFFNG_HOME/fastdkeys"`
 
+* **`server.pruneTime`** Offline Zeit, nach der die Daten für einen Knoten komplett gelöscht werden, z. B.: `"amount": 100, "unit": "days"`
```

Comment:
Beispiel Wert sollte als Objekt angegeben werden, also wie folgt:
`{"amount": 100, "unit": "days"}`

Außerdem sollten dann hier entweder die erlaubten Einheiten aufgelistet werden, oder die Konfiguration der Einfachheit halber nur in Tagen möglich sein.